server/cmd/mmctl/commands/job.go

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/v8/cmd/mmctl/client"
+	"github.com/mattermost/mattermost/server/v8/cmd/mmctl/printer"
+
+	"github.com/spf13/cobra"
+)
+
+var JobCmd = &cobra.Command{
+	Use:   "job",
+	Short: "Management of jobs",
+}
+
+var listJobsCmd = &cobra.Command{
+	Use:     "list",
+	Short:   "List jobs",
+	Example: "  job list",
+	Args:    cobra.NoArgs,
+	RunE:    withClient(listJobsCmdF),
+}
+
+var updateJobCmd = &cobra.Command{
+	Use:     "update [job] [status]",
+	Short:   "Update the status of a job",
+	Example: "  job update pending",
+	Args:    cobra.MinimumNArgs(2),
+	RunE:    withClient(updateJobCmdF),
+}
+
+func init() {
+	listJobsCmd.Flags().Int("page", 0, "Page number to fetch for the list of import jobs")
+	listJobsCmd.Flags().Int("per-page", 5, "Number of import jobs to be fetched")
+	listJobsCmd.Flags().Bool("all", false, "Fetch all import jobs. --page flag will be ignored if provided")
+	listJobsCmd.Flags().StringSlice("ids", nil, "Comma-separated list of job IDs to which the operation will be applied. All other flags are ignored")
+	listJobsCmd.Flags().String("status", "", "Job status")
+	listJobsCmd.Flags().String("type", "", "Job type")
+
+	updateJobCmd.Flags().Bool("force", false, "We generally restrict what statuses you can set but using --force will give you more freedom")
```

Comment:
```suggestion
	updateJobCmd.Flags().Bool("force", false, "It's generally restrict what statuses you can set. You can overwrite these restrictions --force. This might cause undefined behaviour on your Mattermost Server. Use this option with caution.")
```

Reply:
Modified it  a little bit